gradle.properties

```diff
@@ -105,8 +105,8 @@ jetbrains.compose.compiler.version=1.5.14.1
 #   artifactRedirection.version.<redirectionGroup>.<redirectionModule>
```

Comment:
1. Cherry-pick commits between `androidx/compose-ui/1.8.0-rc03` and `androidx/compose-ui/1.8.0` if there are any for `compose` folder (ignore other commits)

2. Wait when the artifact will be available

3. Merge with Merge commit

4. Cherry-pick to `release/1.8`